content/300-guides/500-other/880-troubleshooting-orm/050-creating-bug-reports.mdx

```diff
@@ -120,6 +120,8 @@ Please include any information about your environment and setup. Specifically it
 - Which **version of Prisma** you use (run `prisma -v` to see your Prisma version)
 - Which **version of Node.js** you use (run `node -v` to see your Node.js version)
 
+You can use the [`prisma debug`](/reference/api-reference/command-reference#debug) <span class="api"></span> command to retrieve debugging information.
```

Comment:
Maybe mention that it's only available since `5.6.0` here?
Or alternatively, make it `npx prisma@latest debug`?

Reply:
Because if people have to open that linked page, then they also have to actually scroll to the bottom to find it